src/app/icd/client/DefaultICDClientStorage.cpp

```diff
@@ -465,24 +462,18 @@ CHIP_ERROR DefaultICDClientStorage::DeleteAllEntries(FabricIndex fabricIndex)
     return mpClientInfoStore->SyncDeleteKeyValue(DefaultStorageKeyAllocator::FabricICDClientInfoCounter(fabricIndex).KeyName());
 }
 
-CHIP_ERROR DefaultICDClientStorage::ProcessCheckInPayload(const ByteSpan & payload, ICDClientInfo & clientInfo, bool & refreshKey)
+CHIP_ERROR DefaultICDClientStorage::ProcessCheckInPayload(const ByteSpan & payload, ICDClientInfo & clientInfo, uint32_t & counter)
 {
-    uint32_t counter;
-    MutableByteSpan appData;
+    uint8_t testAppData[32];
```

Comment:
I think size of the array is to big here.

Reply:
just check, resolved.